front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleModal.tsx

```diff
@@ -26,6 +25,8 @@ import {
   TrainScheduleWithPath,
   TrainScheduleWithPathRef,
 } from 'applications/operationalStudies/types';
+import { useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
```

Reply:
Done